utils/run_test.sh

```diff
@@ -200,6 +200,12 @@ if [ -d "/mnt/daos" ]; then
     COMP="UTEST_vea"
     run_test "${SL_PREFIX}/bin/vea_ut"
     run_test "${SL_PREFIX}/bin/vea_stress -d 60"
+    # regression test for DAOS-12256
```

Comment:
probably should set COMP="UTEST_vea_debug" or something so the test gets categorized in Jenkins as a new test